10-get-the-photo/SOLUTION.md

```diff
@@ -1,3 +1,17 @@
 # Solution
 
-TBD
+## Other solutions - [mwhitman189](https://github.com/mwhitman189)
+This seems like a problem screaming for '.reduce', so here it is:
+
+```js
+function getPhotos(arrObject) {
+  const photos = arrObject.reduce((acc, curr) => {
+    acc.push(curr['large']);
+    return acc;
+  }, []);
```

Comment:
```suggestion
    const { large } = curr;
    return [...acc, large ];
  }, []);
```

Reply:
Or you could make it a one liner:

```js
const getPhotos = (photos) => photos.reduce((acc, { large }) => [...acc, large], []);
```

It would actually be nice if you demonstrated how you could go from your first answer to this one liner by refactoring a little each time. Maybe show 3 versions of it.